clappr/clappr-ios
# Fix ready event

### Goal
Adjust the moment when ready event is triggered to be after AVFoundationPlayback is rendered. 

### How to test
1 - Comment the line 597 at file AVFoundationPlayback.swift (with this the video will **not** play).
2 - Run Clappr example app;
3 - Check if ready event appears at log.
